### PR TITLE
[Grouping&Mapping] Package updates and styling fixes

### PR DESCRIPTION
**Pull request by Guillar1**
Styling was starting to break due to class name changes.

### CODE REVIEW

**Review by arnobmallickbsw — APPROVED** [review]
👍

File: packages/itwin/grouping-mapping-widget/package.json [hunk 1]
```diff
@@ -55,33 +55,33 @@
...
-    "@itwin/core-orbitgt": "^3.0.0",
-    "@itwin/core-quantity": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
-    "@itwin/eslint-plugin": "^3.0.0",
-    "@itwin/itwinui-css": "^0.44.1",
-    "@itwin/presentation-common": "^3.0.0",
-    "@itwin/presentation-components": "^3.0.0",
-    "@itwin/presentation-frontend": "^3.0.0",
-    "@itwin/webgl-compatibility": "^3.0.0",
+    "@itwin/appui-abstract": "^3.0.1",
```
Comment (aruniverse) [inline packages/itwin/grouping-mapping-widget/package.json:65]: was there something in 3.0.1 you needed? or would it have worked fine with 3.0.0?

Comment (Guillar1) [reply packages/itwin/grouping-mapping-widget/package.json]: Was in the middle of this actually. Bump unneeded. Thought I had a transitive issue.

**Review by aruniverse — APPROVED** [review]
You guys going to add localization soon? List of unlocalized strings keeps growing...

**Pull request merged**